go-controller/pkg/clustermanager/egressip_controller_test.go

```diff
@@ -4047,11 +4206,32 @@ var _ = ginkgo.Describe("OVN cluster-manager EgressIP Operations", func() {
 				gomega.Eventually(getEgressIP).Should(gomega.Equal(updateEgressIP))
 				_, nodes = getEgressIPStatus(egressIPName)
 				gomega.Expect(nodes[0]).To(gomega.Equal(egressNode2.name))
+				markAfterUpdate := getEgressIPAnnotationValue(egressIPName, util.EgressIPMarkAnnotation)()
+				gomega.Expect(markBeforeUpdate).Should(gomega.Equal(markAfterUpdate), "The mark should be consistent")
+				markInCache, err := fakeClusterManagerOVN.eIPC.getOrAllocMark(egressIPName)
+				gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+				gomega.Expect(markAfterUpdate).Should(gomega.Equal(fmt.Sprintf("%d", markInCache)))
 				return nil
 			}
 
 			err := app.Run([]string{app.Name})
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		})
 	})
+
+	ginkgo.Context("EgressIP Mark cache", func() {
+		ginkgo.It("should round robin when mark range is exhausted", func() {
+			iterations := (util.EgressIPMarkMax - util.EgressIPMarkBase) * 2 // iterate over the allowed range
+			nodeAlloc, _ := getEgressIPMarkAllocator()
```

Comment:
shouldn't we assert an error is not thrown here ?

Reply:
done